solr/core/src/java/org/apache/solr/metrics/AggregateMetric.java

```diff
@@ -51,6 +61,18 @@ public String toString() {
           ", updateCount=" + updateCount +
           '}';
     }
+
+    public double toDouble() {
```

Comment:
Does this need to be `public`?

Reply:
I don't think so, no - and I think some/all of the the other `public` things in these classes could be made less accessible too.  Happy to take a look at this, but would like to make sure the metrics aggregation is working correctly first.

Reply:
Sure, but since this method is new lets make it private from start

Reply:
Yup, made it private.